composer.json

```diff
@@ -11,7 +11,7 @@
         "drupal/console": "~0.10",
         "drupal/core": "~8.1.0",
         "drupal/facets" : "~8.1.0-alpha3",
-        "drupal/og": "dev-ISAICP-2362-DEV",
+        "drupal/og": "dev-update-group-mapping",
```

Comment:
Hmm, you are right about the documentation. I should add the comment in the ticket as well.

I will answer this here and will also add a comment to the update and create a follow up to track this.
The pull request for the main reason of changing branches is https://github.com/amitaibu/og/issues/246.
The ticket is about the 'update group' permission. About the owner of the collection having a permission to edit the group. What I did, as we discussed, I took the latest version of the og and added a small fix for the update group permission until amitaibu responds to it and we fix it properly.